cases/src/org/commcare/cases/ledger/instance/LedgerInstanceTreeElement.java

```diff
@@ -94,10 +94,7 @@ public void attachStringCache(Interner<String> stringCache) {
     public LedgerChildElement getChild(String name, int multiplicity) {
         if ((multiplicity == TreeReference.INDEX_TEMPLATE) &&
                 "ledger".equals(name)) {
-            return null;
-            // TODO PLM: commenting out the following line until http://manage.dimagi.com/default.asp?179998 is fixed
-            // this will break the ledger query silent fail fix for http://manage.dimagi.com/default.asp?177247
-            // return LedgerChildElement.TemplateElement(this);
+            return LedgerChildElement.TemplateElement(this);
```

Comment:
This is an artifact of the spec's structure for ledgers. Since everything after "ledger" is a wildcard, we don't run into the issue where traversing the template passes over TreeElements. This was the behavior during template checking of Ledger/CaseChildElement.